### DISCUSSION TITLE
Send as any - remove on behalf on

### DISCUSSION THREAD

**Discussion by ofekifergan** [postalserver/postal#3163] [Help with using Postal]
Hi all,

I wanted to ask if it possible to remove on behalf on when I activate the send as any feature (add sender header).
I successfully get the mails but I can't figure out how to remove the on behalf.

Any ideas?

**Comment by willpower232** [comment]
I'm not entirely sure you can, it might be worth checking the documentation for the program or website you're reading the email in and seeing if they can explain how it can be removed.

**Comment by ofekifergan** [comment]
Is there any mail I can send a mail without the send as any feature?
Or I should authenticate to any domain I am using

**Reply by willpower232** [reply]
If you're able to add all the domains you are using straight to Postal then you don't need to use the sender header and presumably avoid this issue.

**Reply by Rez0k** [reply]
Thanks, got it
